envFiles/.env.ci

```
@@ -106,10 +105,12 @@ REDIS_MAPPED_PORT=6379
-API_OTEL_ENABLED=true # make it "false" if don't want span logging
-API_OTEL_SERVICE_NAME="talawa-api"
-API_OTEL_ENVIRONMENT="local"
-API_OTEL_SAMPLING_RATIO="1"
-API_OTEL_EXPORTER_OTLP_ENDPOINT="http://localhost:4318/v1/traces" #TODO: after integrating Observability tool
-
-API_SLOW_REQUEST_MS=500
\ No newline at end of file
+API_OTEL_ENABLED=false
+API_OTEL_ENVIRONMENT=production
+# Container-to-container
+API_OTEL_EXPORTER_OTLP_ENDPOINT="http://jaeger:4318/v1/traces"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)